ehrql/loaders.py

```diff
@@ -59,6 +88,69 @@ def load_definition_in_subprocess(definition_type, definition_file, user_args, e
     return deserialize(result.stdout)
 
 
+def isolation_is_supported():
+    return subprocess.run([PLEDGE_BIN, "-T", "pledge"]).returncode == 0
+
+
+def subprocess_run_isolated(args, **kwargs):
+    return subprocess.run(
+        [
+            PLEDGE_BIN,
+            # Configure "pledge" to restrict avilable operations to the minimum needed
+            # to execute a definition file and serialize the result, which are:
+            # - stdio: allow stdio and benign system calls
```

Comment:
Although I see this is just copied from the Oledge docs, and we've already decided we trust Pledge enough to use it, the handwaviness here makes me feel a bit uneasy. I get why stdio would be considered benign, but what else lurks there?

There's nothing we can do about this, realistically, but _possibly_ worth linking to the [part of the Pledge docs](https://justine.lol/pledge/#promises) that actually describes what the calls are?